tag/reader.py

```diff
@@ -143,16 +146,17 @@ def __iter__(self):
                 self.inferred_regions[feature.seqid].start = min(rstr, fstr)
                 self.inferred_regions[feature.seqid].end = max(rend, fend)
 
+                featureid = feature.get_attribute('ID')
                 parentid = feature.get_attribute('Parent')
                 if parentid is None:
-                    # All components of a multi-feature are added here...
-                    self.records.append(feature)
+                    # Only add one entry from each multi-feature
+                    if featureid is None or featureid not in self.featsbyid:
+                        self.records.append(feature)
```

Comment:
Previously, the reader used to track all components of a multi-feature. Now, just the rep.